api/src/main/java/org/pmiops/workbench/db/model/DbUser.java

```diff
@@ -92,6 +94,8 @@
 
   private Timestamp publicationsLastConfirmedTime;
 
+  @VisibleForTesting static final int PROJECT_BILLING_ID_SIZE = 8;
```

Comment:
`PROJECT_BILLING` seems to be a copy paste error.
Also 8 is bit too long(project choose 8 because that name is global unique). 3~4 for PD is enough.